ansible/roles/kubernetes/templates/config.j2

```diff
@@ -20,4 +20,4 @@ KUBE_LOG_LEVEL="--v=0"
 KUBE_ALLOW_PRIV="--allow-privileged=true"
 
 # How the replication controller, scheduler, and proxy
-KUBE_MASTER="--master=https://{{ groups['masters'][0] }}:{{ kube_master_api_port }}"
+KUBE_MASTER="--master=https://{{ kube_apiserver_endpoint }}:{{ kube_master_api_port }}"
```

Comment:
Why can't we use the following instead of introducing a new kube_apiserver_endpoint param?

Ex>
```
KUBE_MASTER="--master=https://{% if groups['masters']|length > 1 %}{{ kube_apiserver_vip }}{% else %}{{ groups['masters'][0] }}{% endif %}:{{ kube_master_api_port }}"
```

We really want to do everything possible to minimize new top-level vars. I think ```kube_apiserver_vip``` make perfect sense, but I still feel we can get rid of ```kube_apiserver_endpoint```